options/locale/locale_en-US.ini

```diff
@@ -1505,9 +1505,10 @@ issues.due_date_form_edit = "Edit"
 issues.due_date_form_remove = "Remove"
 issues.due_date_not_writer = "You need repository write access to update an issue's due date."
 issues.due_date_not_set = "No due date set."
-issues.due_date_added = "added the due date %s %s"
-issues.due_date_modified = "modified the due date from %[2]s to %[1]s %[3]s"
-issues.due_date_remove = "removed the due date %s %s"
+issues.due_date_added = "added the due date"
+issues.due_date_modified_from = "modified the due date from"
+issues.due_date_modified_to = "to"
+issues.due_date_remove = "removed the due date"
```

Comment:
It's better not to do so. At least, wait for a proper i18n system (with correct datetime support)

* #23863